addons/hr_attendance/models/hr_attendance.py

```diff
@@ -168,7 +168,10 @@ def _update_overtime(self, employee_attendance_dates=None):
             )[emp.resource_id.id]
             # Substract Global Leaves and Employee's Leaves
             leave_intervals = emp.resource_calendar_id._leave_intervals_batch(
-                start, stop, emp.resource_id, domain=self._get_overtime_leave_domain()
+                start, stop, emp.resource_id, domain=AND([
+                    self._get_overtime_leave_domain(),
+                    [('company_id', 'in', [False, emp.company_id.id])],
```

Comment:
I don't think there's a point in adding False in the domain here, because in the function `_leave_intervals_batch` , if the company_id of the resource.calendar.leaves is False, we will never pass this condition : 
`if leave.resource_id.id not in [False, resource.id] or (not leave.resource_id and resource and resource.company_id.id != leave.company_id.id):`, 
so we should either modify this condition like this : 
`if leave.resource_id.id not in [False, resource.id] or (not leave.resource_id and resource and (leave.company_id and resource.company_id.id != leave.company_id.id)):`
or remove False from the domain.

Reply:
why never ? resources contains an empty record resource.resource to compute the global holidays and then used in line 176 in the same file.

Reply:
friday brain my bad you're right

Reply:
haha @jugj-odoo it happens